docs/docker-compose/start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+git clone https://github.com/ceph/paddles.git
+cd paddles
+git fetch origin pull/94/head:wip-amathuria-removing-beanstalkd
+git checkout wip-amathuria-removing-beanstalkd
+cd ../
+git clone https://github.com/ceph/pulpito.git
+git clone https://github.com/ceph/teuthology.git
+cd teuthology
+git fetch origin pull/1650/head:wip-amathuria-replace-beanstalkd-paddles
+git checkout wip-amathuria-replace-beanstalkd-paddles
+docker-compose up
```

Comment:
```suggestion
docker-compose up --build
```

Reply:
@kamoltat this got things working for me. there are tons of errors after things come up, but that may be unrelated. for reference, they seem to be related to:

```
postgres_1   | 2021-08-11 21:51:54.922 UTC [114] ERROR:  relation "job_id_sequence" does not exist at character 16
```

Reply:
@amathuria This is the error I was telling you about, think we might need to change `job_id_sequence` to what is saved in psql: `jobs_id_seq`.

Reply:
Ah okay I see. I'm trying this out now so I'll let you know how we can fix it.

Reply:
It's fixed now.

Reply:
thank you!